src/popsift/sift_pyramid.h

```diff
@@ -152,8 +151,8 @@ class Pyramid
 
     void descriptors( const Config& conf );
 
-    void debug_out_floats  ( float* data, uint32_t pitch, uint32_t height );
-    void debug_out_floats_t( float* data, uint32_t pitch, uint32_t height );
+//    void debug_out_floats  ( float* data, uint32_t pitch, uint32_t height );
```

Comment:
@griwodz I commented out these methods because they are not implemented, shall we remove them?

Reply:
Sorry about the delay. Yes, please remove them!